### PR TITLE
Updates to action servers and the BT with dynamic dynamic reconfigure

### PR DESCRIPTION
**Pull request by svbhat**
Updating changes from Özer and Sriharsha to enable the BT to send setpoints to sam_basic_controllers.

### CODE REVIEW

**Comment by svbhat** [comment]
@KKalem PR made for smarc_missions, I will also make PRs for the updated controllers and sam_stonefish_sim later this week or next week.

**Comment by nilsbore** [comment]
@KKalem Please check that this is OK!

**Comment by nilsbore** [comment]
@KKalem or @svbhat Can either of you try running these changes purely from an install space, with no other `smarc_missions` sourced or installed?

**Comment by KKalem** [comment]
> 
> 
> @KKalem or @svbhat Can either of you try running these changes purely from an install space, with no other `smarc_missions` sourced or installed?

Isn't this exactly what CI is testing? Or am I missing something?

**Comment by nilsbore** [comment]
Yes, I'm not sure if it does. Please try running it from an install space.

**Comment by svbhat** [comment]
> Yes, I'm not sure if it does. Please try running it from an install space.

I had initially run it from a clean install space and it worked during development. I will try running this again from a new install space early next week.

**Comment by nilsbore** [comment]
@KKalem @svbhat Why are we including `ddynamic_reconfigure` as a package within this repo. As far as I can tell, both `ddynamic_reconfigure` and `ddynamic_reconfigure_python` is released for melodic [1](https://index.ros.org/p/ddynamic_reconfigure/#melodic), [2](http://wiki.ros.org/ddynamic_reconfigure_python) and noetic [1](https://index.ros.org/p/ddynamic_reconfigure/#noetic), [2](http://wiki.ros.org/ddynamic_reconfigure_python). You should only need to specify the dep in the `package.xml` to get it with rosdep.

**Comment by nilsbore** [comment]
If there is no need to include it here, please perform a tree filter to remove it completely from the git history, then force-push to your PR branch.

File: smarc_missions.rosinstall [hunk 1]
```diff
@@ -5,3 +5,5 @@
 - git: {local-name: lolo_common, uri: 'https://github.com/smarc-project/lolo_common.git', version: noetic-devel}
 - git: {local-name: sbg_ros_driver, uri: 'https://github.com/smarc-project/sbg_ros_driver.git', version: master}
 - git: {local-name: geographic_info, uri: 'https://github.com/ros-geographic-info/geographic_info.git', version: master}
+- git: {local-name: ddynamic_reconfigure_python, uri: 'https://github.com/pal-robotics/ddynamic_reconfigure_python.git', version: master}
```
Comment (nilsbore) [inline smarc_missions.rosinstall]: This should be removed also

**Comment by nilsbore** [comment]
I'm waiting for @KKalem to do the tree filter and force push it to this branch before merging.

**Comment by nilsbore** [comment]
OK, tried this in the simulator and it works! Nice job!

**Pull request merged**